### PR TITLE
[Benchmark] add paddleSeg G0 G1 G3 model for benchmark

### PR DESCRIPTION
**Pull request by Stinky-Tofu**
**[Benchmark-paddleSeg]** Add "ocrnet_hrnetw48" "fastscnn" "stdc_stdc1" "bisenetv2" "fcn_hrnetw18" "pphumanseg_lite" "ppmatting" "pp_liteseg_stdc1" "pp_liteseg_stdc2" for benchmark

03dfc268d66c3e0977c2776974b0d68f
11c96146deda81b0d564acf065389dfd
27aae5dd6fc46eda69a9dd97ae2b704f
390c0653d2842a592d548e348910786b
63493c40ac449d7196da6c3347aab68b
ce7ff20f7e64553b5a62cd229bd54780

### CODE REVIEW

File: frame_benchmark/docker_images.yaml [hunk 1]
```diff
@@ -30,23 +30,26 @@ pytorch:
...
+   fcos_r50_fpn_1x_coco: iregistry.baidu-int.com/paddle-benchmark/paddlecloud-base-image:paddlecloud-ubuntu18.04-gcc8.2-cuda10.2-cudnn7
    faster_rcnn_r50_fpn_1x_coco: iregistry.baidu-int.com/paddle-benchmark/paddlecloud-base-image:paddlecloud-ubuntu18.04-gcc8.2-cuda11.2-cudnn8
    deformable_detr_r50_1x_coco: iregistry.baidu-int.com/paddle-benchmark/paddlecloud-base-image:paddlecloud-ubuntu18.04-gcc8.2-cuda11.2-cudnn8
    mask_rcnn_r50_1x_coco: iregistry.baidu-int.com/paddle-benchmark/paddlecloud-base-image:paddlecloud-ubuntu18.04-gcc8.2-cuda10.2-cudnn7
-   mask_rcnn_r50_fpn_1x_coco: iregistry.baidu-int.com/paddlecloud/base-images:paddlecloud-ubuntu18.04-gcc8.2-cuda11.7-cudnn8.4.1-nccl2.12.12 
-   yolov3_darknet53_270e_coco: iregistry.baidu-int.com/paddlecloud/base-images:paddlecloud-ubuntu18.04-gcc8.2-cuda11.7-cudnn8.4.1-nccl2.12.12 
+   mask_rcnn_r50_fpn_1x_coco: iregistry.baidu-int.com/paddle-benchmark/paddlecloud-base-image:paddlecloud-ubuntu18.04-gcc8.2-cuda11.2-cudnn8
+   yolov3_darknet53_270e_coco: iregistry.baidu-int.com/paddle-benchmark/paddlecloud-base-image:paddlecloud-ubuntu18.04-gcc8.2-cuda11.2-cudnn8
```
Comment (mmglove) [inline frame_benchmark/docker_images.yaml:38]: PR中修改了其他模型库的内容，请check

Comment (Stinky-Tofu) [reply frame_benchmark/docker_images.yaml]: 已修复 @mmglove

File: frame_benchmark/pytorch/dynamic/PaddleSeg/scripts/bisenetv2/benchmark_common/prepare.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+echo "******* install enviroments for benchmark ***********"
+echo `pip --version`
+
+if [ ! -f "torch_dev_whls.tar" ];then
+  wget https://paddle-wheel.bj.bcebos.com/benchmark/torch_dev_whls.tar
```
Comment (mmglove) [inline frame_benchmark/pytorch/dynamic/PaddleSeg/scripts/bisenetv2/benchmark_common/prepare.sh:7]: 参考添加规范，引入 FLAG_TORCH_WHL_URL 作为torch_dev_whls.tar 的下载链接

File: frame_benchmark/pytorch/dynamic/PaddleSeg/scripts/fastscnn/N4C32/fastscnn_bs4_fp32_DP.sh
Comment (mmglove) [inline frame_benchmark/pytorch/dynamic/PaddleSeg/scripts/fastscnn/N4C32/fastscnn_bs4_fp32_DP.sh:1]: 请不要删除多机脚本

**Pull request closed without merging**